specification/cognitiveservices/AnomalyDetector/multivariate/models.tsp

```diff
@@ -8,6 +8,7 @@ using Azure.Core;
 
 namespace AnomalyDetector.Multivariate;
 
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "Legacy spec"
```

Comment:
Why do you need to suppress `documentation-required` when there is `@doc` on the following line?  Is it because each enum value is also supposed to have `@doc`?

Reply:
Yes, the scope of the suppression is for the entire enumeration